src/v/redpanda/admin_server.h

```diff
@@ -40,6 +40,15 @@ struct admin_server_cfg {
     ss::scheduling_group sg;
 };
 
+enum class debug_bundle_status { initialized, in_progress, finished };
+
+struct debug_bundle {
+    static constexpr ss::shard_id debug_bundle_shard_id = 0;
+    debug_bundle_status status;
+    ss::sstring filename;
+    ss::sstring rpk_pid;
+};
+
```

Comment:
Consider making this a sharded service.

Reply:
>This is not a POC. We are experimenting

What kind of feedback are you looking for? The cover letter seems like we are testing the waters with the seastar process utility.

>Consider making this a sharded service.

But this comment makes it sound you are reviewing the design itself?

Reply:
> > This is not a POC. We are experimenting
> 
> What kind of feedback are you looking for? The cover letter seems like we are testing the waters with the seastar process utility.

Correct, we are testing seastars process utility here. I'm looking for feedback on potential pitfalls or problems from spawning a process via Redpanda.

> > Consider making this a sharded service.
> 
> But this comment makes it sound you are reviewing the design itself?

Yes I am reviewing the design async to the above feedback. Going forward, I'll leave my personal notes out

Reply:
> Going forward, I'll leave my personal notes out

no worries at all just wanted to make sure I was providing the feedback you were looking for